kDrive/AppDelegate.swift

```diff
@@ -233,9 +233,12 @@ final class AppDelegate: UIResponder, UIApplicationDelegate, AccountManagerDeleg
 
         let currentState = RootViewControllerState.getCurrentState()
         prepareRootViewController(currentState: currentState)
-        if case .mainViewController = currentState {
+        switch currentState {
+        case .mainViewController, .appLock:
```

Comment:
This is not called at startup.

Reply:
Maybe we refactor this code so that it's called both in `applicationWillEnterForeground` and in `willFinishLaunchingWithOptions` right after `prepareRootViewController`

Reply:
Yep exactly what I did 👍